src/main/java/com/health/controller/HomeController.java

```diff
@@ -930,27 +933,23 @@ public String viewCoursesAvailable(HttpServletRequest req,
         
         if(typeTutorial!=null && !typeTutorial.isEmpty() && typeTutorial.equals("typeTutorial")) {
         	typeTutorial=CommonData.DOCUMENT_TYPE_TUTORIAL_ORIGINAL_SCRIPT;
-        }else {
-			typeTutorial="";
-		}
+        	 model.addAttribute("typeTutorialforQuery", "Tutorial");
+        }
         
         if(typeTimeScript!=null && !typeTimeScript.isEmpty() && typeTimeScript.equals("typeTimeScript")) {
         	typeTimeScript=CommonData.DOCUMENT_TYPE_TUTORIAL_TIME_SCRIPT;
-        }else {
-        	typeTimeScript="";
-		}
+        	 model.addAttribute("typeTimeScriptforQuery", "Timed Script");
+        }
         
         if(typeBrochure!=null && !typeBrochure.isEmpty() && typeBrochure.equals("typeBrochure")) {
         	typeBrochure=CommonData.DOCUMENT_TYPE_BROCHURE;
-        }else {
-        	typeBrochure="";
-		}
+        	 model.addAttribute("typeBrochureforQuery", "Promotional Material ");

```

Comment:
```suggestion
        	 model.addAttribute("typeBrochureforQuery", "Promotional Material");
```
Remove space on the end.